snmp/assets/configuration/spec.yaml

```diff
@@ -21,6 +21,17 @@ files:
           example: core
           type: string
           display_default: python
+      - name: use_device_id_as_hostname
+        enabled: true
+        description: |
+            Use `device:<DEVICE_ID>` (device_id is composed of `<NAMESPACE>:<DEVICE_IP_ADDRESS>`) as `hostname`
```

Comment:
What about being explicit and saying 
```
Use `device:<NAMESPACE><DEVICE_IP_ADDRESS>` 
```
?

Reply:
Since the option is about `device_id` (use_device_id_as_hostname), I favoured `device:<DEVICE_ID>` :)